scripts/release/README.md

```diff
@@ -26,13 +27,13 @@ If this is your first time running the release scripts, go to the `scripts/relea
 To prepare a build for a particular commit:
 1. Choose a commit from [the commit log](https://github.com/facebook/react/commits/master).
 2. Click the "“✓" icon and click the Circle CI "Details" link.
-3. Select the `build` job (**not** the `build_experimental` job; see the next section).
+3. Select the `process_artifacts ` job (**not** the `process_artifacts_experimental`job; see the next section)
```

Comment:
Really the only thing that matters is which workflow it's from because of this part:

https://github.com/facebook/react/blob/af1b039bdd5a8b5def5d51acad00b79e9b7b377c/scripts/release/utils.js#L44-L56

So the previous instructions happened to also work

Reply:
Right. The reason I changed the target is because the new display strings made the old targets hard to find. (They got truncated in the popup.) :smile: